hw9/src/Main.java

```diff
@@ -0,0 +1,113 @@
+/*
+Задача 2
+Также бухгалтерия попросила найти минимальную и максимальную трату за день.
+Напишите программу, которая решит эту задачу, и выведите в консоль результат в формате:
+«Минимальная сумма трат за день составила … рублей. Максимальная сумма трат за день составила … рублей».
+*/
+        task2();
+        System.out.println(" ");
+/*
+Задача № 3
+Теперь бухгалтерия хочет понять, какую в среднем сумму компания тратила в течение 30 дней.
+Напишите программу, которая посчитает среднее значение трат за месяц (то есть сумму всех трат за месяц поделить
+на количество дней), и выведите в консоль результат в формате: «Средняя сумма трат за месяц составила … рублей».
+Важно помнить: подсчет среднего значения может иметь остаток, то есть быть не целым, а дробным числом.
+*/
+        System.out.println("Задача №3\n");
+        task3();
+        System.out.println();
+/*
+Задача № 4
+В бухгалтерской книге появился баг. Что-то пошло не так: фамилии и имена сотрудников начали отображаться в обратную сторону.
+Т. е. вместо «Иванов Иван» мы имеем «навИ вонавИ». Данные с именами сотрудников хранятся в виде массива символов char[]
+Напишите код, который в случае такого бага будет выводить фамилии и имена сотрудников в корректном виде.
+В качестве данных для массива используйте: char[] reverseFullName = { 'n', 'a', 'v', 'I', ' ', 'v', 'o', 'n', 'a', 'v', 'I'};
+В результате в консоль должно быть выведено: Ivanov Ivan.
+Важно: не используйте дополнительные массивы для решения этой задачи. Необходимо корректно пройти по массиву циклом и распечатать его элементы в правильном порядке.
+*/
+        System.out.println("Задача №4\n");
+        task4();
+    }
+
+    public static int[] generateRandomArray() {
+        Random random = new Random();
+        int[] arr = new int[30];
+        for (
+                int i = 0;
+                i < arr.length; i++) {
+            arr[i] = random.nextInt(100_000) + 100_000;
+        }
+        return arr;
+    }
+
+    public static void task1() {
+        int[] arr = generateRandomArray();
+        for (int i = 0; i < arr.length; i++) {
+            System.out.printf("Сумма трат за месяц составила %d рублей\n", arr[i]);
+        }
+    }
+
+    public static void task2() {
+        int[] arr = generateRandomArray();
+        int n = arr.length;
+        for (int i = 0; i < n - 1; i++) {
+            for (int j = 0; j < n - i - 1; j++) {
+                if (arr[j] > arr[j + 1]) {
+                    // Обмен элементов
+                    int temp = arr[j];
+                    arr[j] = arr[j + 1];
+                    arr[j + 1] = temp;
+                }
+            }
+        }
+        System.out.printf("Минимальная сумма трат за день составила %d рублей\n", arr[0]);
+        System.out.printf("Максимальная сумма трат за день составила %d рублей\n", arr[29]);
+    }
+
+    public static void task3() {
+        int[] arr = generateRandomArray();
+        int sum = 0;
+        int total = 30;
```

Comment:
```suggestion
        int total = arr.length;
```
нужно использовать длину массива